BlueprintUI/Sources/ViewDescription/LifecycleCallback.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+/// This is the type used by all lifecycle callback hooks.
+///
+/// - Tag: ElementLifecycle
+///
+/// # The Element Lifecycle
+///
+/// View-backed elements have a lifecycle with 3 states:
+///
+/// 1. **Unmounted**
+///
+///    This is the initial state of an element, before a render pass.
+///
+/// 2. **Mounted**
```

Comment:
When would hooking into the mount/unmount events be useful in practice?

Reply:
> When would hooking into the mount/unmount events be useful in practice?

I don't have any uses in mind! Mount/unmount were a lot easier to write than appear/disappear, and I left them in for completeness.